README.md

```diff
@@ -557,28 +529,56 @@ To give the charm access to some containers, you need to pass them to the input
 An example of a state including some containers:
 
 ```python
-from scenario.state import Container, State
+import scenario
+
+# A dictionary? This means there's duplication of the name, and nothing really
+# stopping you using a different state name than the actual container name, which is weird.
```

Comment:
True, though the consistency checker could verify this.

Of the options shown here, I definitely prefer this one, the dumb dict (even though there's a little bit of duplication), as it seems it would result in the simplest types and clear tests.

That said, does the container need a `.name` attribute? Or what if we had `run`, or even `scenario.State()`, fill in the `.name` attribute from the dict key?

state = scenario.State(containers={
    "foo": scenario.Container(name="foo", can_connect=True),
    "bar": scenario.Container(name="bar", can_connect=False)
})